### ISSUE TITLE
Добавить в схему MarketInstrument поля начала и окончания торгов

### ISSUE THREAD

**Issue by vladimir1980** [TinkoffCreditSystems/invest-openapi#524]
1. Сейчас у нас реализована синхронизация всех инструментов (данные и свечи) в postgres, данные забираются из сокета и агрегируются, недостающие или потеряные данные забираются по ресту.
2. Синхронизация по ресту подразумевает вычисление диффа между локальной базой и рест данными.
3. Сейчас нет различия между _"инструмент не торговался"_ и _"данные за временнОй период инструмента потерялись"_
4. **Проблема** - большая часть времени синрхронизации тратится на порядка **20 тысяч пустых запросов по ресту** , с учетом рейт лимитов, процесс происходит неприемлемо долго. 
4. **Добавление полей начала и окончания торгов** решило бы это проблему при составлении запроса на расхождение.

**Comment by vladimir1980** [comment]
В идеале отдельная сущность, **расписание торгов инструмента**

**Comment by AlexanderVolkovTCS** [comment]
Да, запланирован метод, возвращающий время начала и завершения торговых сессий.

**Comment by QiXi** [comment]
+1 ожидаю. Также напишу тут что `StreamingTradeStatus.normalTrading` не приходит тогда, кода на самом деле торги открыты и получается бесполезным индикатором. Например торги `SPCE` начинаются в `7:00`, а статус меняется в `7:18`
